lib/src/content/components/paginating_group/paginating_widgets_group.dart

```diff
@@ -2,11 +2,11 @@ import 'package:flutter/material.dart';
 import 'package:wolt_modal_sheet/src/content/components/current/current_main_content_animated_builder.dart';
 import 'package:wolt_modal_sheet/src/content/components/current/current_sab_animated_builder.dart';
 import 'package:wolt_modal_sheet/src/content/components/current/current_top_bar_animated_builder.dart';
-import 'package:wolt_modal_sheet/src/content/components/current/current_top_bar_controls_animated_builder.dart';
+import 'package:wolt_modal_sheet/src/content/components/current/current_navigation_toolbar_animated_builder.dart';
 import 'package:wolt_modal_sheet/src/content/components/outgoing/outgoing_main_content_animated_builder.dart';
 import 'package:wolt_modal_sheet/src/content/components/outgoing/outgoing_sab_animated_builder.dart';
 import 'package:wolt_modal_sheet/src/content/components/outgoing/outgoing_top_bar_animated_builder.dart';
-import 'package:wolt_modal_sheet/src/content/components/outgoing/outgoing_top_bar_controls_animated_builder.dart';
+import 'package:wolt_modal_sheet/src/content/components/outgoing/outgoing_navigation_toolbar_animated_builder.dart';
```

Comment:
In the future we should streamline the imports, this is getting very long and wide.

Reply:
You should have access to create a card in this board. https://github.com/orgs/woltapp/projects/1/views/1 Can you elaborate your suggestion or specs in the board as card or issue so that someone else can contribute?